### PR TITLE
feat: add icons for exchanges

### PR DESCRIPTION
**Pull request by alfonsobries**
## Summary

Related to https://app.clickup.com/t/861mruhvw

## Checklist

- [ ] Documentation _(if necessary)_
- [ ] Tests _(if necessary)_
- [ ] Ready to be merged

### CODE REVIEW

**Review by alexbarnsley — APPROVED** [review]

**Review by alexbarnsley — CHANGES_REQUESTED** [review]
the id's have been overwritten. e.g. `arkscan/icons/bingx.svg` has `id="a"`. you'll need to replace all these instances with `id="keep_whatevertheidis"`. if they all use the same they'll mess with each other when on the same page

File: arkscan/icons/letsexchange.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...34-.012.07.019.189.19.189h2.902c.212 0 .164.185.063.315-.1.13-5.058 8.173-5.11 8.26-.052.088-.287.102-.253-.126.035-.227.684-5.149.694-5.233.011-.085.013-.252-.189-.252H6.997c-.101 0-.195-.094-.063-.316.131-.222 5.012-8.05 5.11-8.197.098-.146.264.025.252.126Z" fill="#C3F166"/><path d="M7.084 17.559a7.574 7.574 0 0 1-6.852-5.692A7.563 7.563 0 0 1 3.544 3.6a7.574 7.574 0 0 1 6.105-.87 75.84 75.84 0 0 0-.976 1.546 5.84 5.84 0 0 0-6.15 2.829 5.83 5.83 0 0 0 4.892 8.732c-.044.162-.28 1.47-.33 1.722Z" fill="url(#a)"/><path d="M13.736 4.955a5.05 5.05 0 0 1 5.848 2.918 5.043 5.043 0 0 1-4.159 7.001 5.05 5.05 0 0 1-2.325-.324c.265-.45.657-.963.777-1.216a3.632 3.632 0 0 0 4.547-4.553 3.627 3.627 0 0 0-3.261-2.534 3.632 3.632 0 0 0-1.71.322 30.8 30.8 0 0 1 .283-1.614Z" fill="url(#b)"/><defs><linearGradient id="a" x1="6.844" y1="2.047" x2="6.819" y2="17.559" gradientUnits="userSpaceOnUse"><stop stop-color="#667180"/><stop offset="1" stop-color="#fff" stop-opacity="0"/></linearGradient><linearGradient id="b" x1="14.245" y1="14.882" x2="14.363" y2="4.803" gradientUnits="userSpaceOnUse"><stop stop-color="#667180"/><stop offset="1" stop-color="#fff" stop-opacity="0"/></linearGradient></defs></svg>
```
Comment (ItsANameToo) [inline arkscan/icons/letsexchange.svg]: id

File: arkscan/icons/bingx.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg viewBox="0 0 20 17" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M8.615 5.751C5.465 10.748 1.744 11.446 0 11.342v4.66c2.872-.311 8.153-1.398 11.384-6.368 2.827-4.349 6.461-5.436 8.307-5.436V.005c-2.564-.052-7.845.621-11.076 5.746Z" fill="url(#a)"/><path d="M11.385 5.695c3.15 4.948 6.871 5.64 8.615 5.538v4.615c-2.872-.308-8.153-1.385-11.384-6.308C5.789 5.233 2.155 4.156.309 4.156V.003C2.873-.05 8.154.618 11.385 5.695Z" fill="#2A66FF"/><defs><linearGradient id="a" x1="20.153" y1="1.848" x2=".308" y2="14.463" gradientUnits="userSpaceOnUse"><stop stop-color="#2A66FF"/><stop offset=".505" stop-color="#174EDB"/><stop offset="1" stop-color="#2A66FF"/></linearGradient></defs></svg>
```
Comment (ItsANameToo) [inline arkscan/icons/bingx.svg]: id

File: arkscan/icons/bitexen.svg [hunk 3]
```diff
@@ -0,0 +1 @@
+<svg viewBox="0 0 19 14" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M1.85 14s8.052-8.067 9.105-9.105c1.053-1.039 2.988-1.31 3.983-.285.996 1.025 1.238 2.775.427 3.7-.81.924-1.138 1.294-1.422 1.565-.285.27-.811.74-1.537.128-.256-.214-.427-.37-.427-.37l-2.305 1.664s1.252 1.238 1.836 1.878c.583.64 1.806 1.039 2.717.256.91-.782 1.508-1.38 2.846-2.845 1.337-1.466 2.945-5.293 0-7.825-2.946-2.533-6.275-1.708-8.11.142C7.128 4.753.825 11.027.569 11.297c-.256.27-.441 1.01 0 1.423.44.413 1.28 1.28 1.28 1.28Z" fill="#53B7EF"/><path d="M0 1.48S.726.74 1.138.342c.413-.399 1.11-.512 1.565 0 .456.512 9.533 9.532 9.533 9.532l-1.85 2.134L0 1.48Z" fill="url(#a)"/><defs><linearGradient id="a" x1="5.5" y1="4.5" x2="10" y2="9.5" gradientUnits="userSpaceOnUse"><stop stop-color="#4496F7"/><stop offset="1" stop-color="#53B7EF"/></linearGradient></defs></svg>
```
Comment (ItsANameToo) [inline arkscan/icons/bitexen.svg]: id

File: arkscan/icons/coinex.svg [hunk 4]
```diff
@@ -0,0 +1 @@
+<svg viewBox="0 0 19 21" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M12.724 7.89a3.833 3.833 0 0 0-5.45 0c-1.508 1.533-1.492 4.03.015 5.563a3.823 3.823 0 0 0 5.467.016c.025-.026.131-.145.318-.355l-1.818-1.852c-.187.221-.285.337-.296.347a1.3 1.3 0 0 1-1.86 0c-.512-.522-.512-1.37-.015-1.876a1.295 1.295 0 0 1 1.843.017l4.86 4.94c-1.33 1.958-3.561 3.2-5.966 3.2-3.976 0-7.214-3.312-7.214-7.39 0-4.078 3.238-7.39 7.214-7.39 2.325 0 4.457 1.142 5.788 2.985-.215.254-.763.89-1.644 1.907l1.793 1.82 2.977-3.417C17.196 2.85 13.718.5 9.822.5 4.42.5.043 4.986.043 10.5s4.377 10 9.78 10c2.452 0 4.488-.783 6.476-2.512 1.811-1.566 2.663-3.738 2.663-3.738l-6.238-6.36Z" fill="url(#a)"/><defs><linearGradient id="a" x1="18.962" y1="10.5" x2=".043" y2="10.5" gradientUnits="userSpaceOnUse"><stop stop-color="#22E6B8"/><stop offset="1" stop-color="#00C1CE"/></linearGradient></defs></svg>
```
Comment (ItsANameToo) [inline arkscan/icons/coinex.svg]: id

File: arkscan/icons/coinspot.svg [hunk 5]
```diff
@@ -0,0 +1 @@
...-.021-.027-.043.025-.07.07-.128.107-.191.12-.198.244-.393.375-.583l.154-.221c.012-.017.027-.034.025-.058.094.05.181.113.27.172Zm.23 1.637c-.007-.035.023-.053.04-.075.218-.31.422-.63.616-.955l.015-.02c.092.054.181.113.268.174-.029.015-.039.045-.054.07-.154.258-.317.51-.486.758l-.15.21c-.017.01-.032.003-.046-.006l-.175-.112c-.014-.011-.029-.023-.028-.044Z" fill="#637282"/><path d="m10.626 10.776.25.16.28.19c-.02.004-.028.022-.038.037a9.024 9.024 0 0 1-.936 1.204c-.072.076-.143.153-.219.226a4.351 4.351 0 0 1-.782.623 1.592 1.592 0 0 1-.49.204c-.157.032-.307.02-.436-.084a.535.535 0 0 1-.18-.331 1.939 1.939 0 0 1 .024-.728c.081-.41.225-.8.395-1.181.188-.422.412-.824.662-1.212.01-.014.02-.027.018-.045l.238.152.052.028c.083.063.172.115.26.17-.225.385-.43.778-.588 1.196a1.93 1.93 0 0 0-.146.589.326.326 0 0 0 .023.15.128.128 0 0 0 .151.079c.14-.027.25-.11.355-.195.242-.198.452-.427.652-.667a9.36 9.36 0 0 0 .416-.536c.011-.014.02-.028.04-.03Z" fill="#EBBF4B"/><path d="M14.238 3.081a.933.933 0 0 1 .965.866.936.936 0 0 1-.882 1.008.942.942 0 0 1-.985-.856c-.036-.55.346-.982.902-1.018Z" fill="#fff"/></g><defs><clipPath id="a"><path fill="#fff" d="M0 0h20.008v20.008H0z"/></clipPath></defs></svg>
```
Comment (ItsANameToo) [inline arkscan/icons/coinspot.svg]: id

File: arkscan/icons/exodus.svg [hunk 6]
```diff
@@ -0,0 +1 @@
+<svg viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="m14 9.984 6 5.991h-2.125c-.375 0-.688-.125-.938-.374l-5.562-5.617 5.563-5.553c.25-.25.625-.375.937-.375H20l-6 5.928ZM3.062 4.431a1.38 1.38 0 0 0-.937-.375H0l6 5.928-6 5.991h2.125c.375 0 .688-.125.938-.374l5.562-5.554c0-.063-5.563-5.616-5.563-5.616ZM10 11.357l-5.563 5.554c-.25.25-.375.624-.375.936v2.122l6-5.99 6 5.99v-2.122a1.27 1.27 0 0 0-.375-.936L10 11.357Zm0-2.745 5.563-5.554c.25-.25.374-.624.374-.936V0L10 5.99 4 0v2.122c0 .374.125.686.375.936L10 8.612Z" fill="url(#a)"/><defs><linearGradient id="a" x1="-.006" y1="9.984" x2="20.009" y2="9.984" gradientUnits="userSpaceOnUse"><stop stop-color="#00BFFF"/><stop offset="1" stop-color="#6619FF"/></linearGradient></defs></svg>
```
Comment (ItsANameToo) [inline arkscan/icons/exodus.svg]: id

File: arkscan/icons/bitladon.svg [hunk 7]
```diff
@@ -0,0 +1 @@
...p://www.w3.org/2000/svg"><g clip-path="url(#a)" fill="#454FBC"><path d="M11.624 9.917h.125a.791.791 0 0 0 .792-.792v-.75a.791.791 0 0 0-.792-.792H11V6.375h-.708v1.208h-.917V6.375h-.708v1.208H7.583a.18.18 0 0 0-.167.167v4.417a.18.18 0 0 0 .167.166h1.083v1.084h.708v-1.084h.917v1.084h.708v-1.084h.625a.943.943 0 0 0 .959-.958v-.542c.041-.5-.417-.916-.959-.916ZM8.458 8.375h2.75c.208 0 .375.167.375.375v.417a.373.373 0 0 1-.375.375h-2.75V8.375Zm3.166 2.792c0 .25-.166.416-.416.416h-2.75v-1.25h2.75c.25 0 .416.167.416.417v.417Z"/><path d="M10 0v.708c5.125 0 9.292 4.167 9.292 9.292S15.125 19.292 10 19.292.708 15.125.708 10H0c0 5.542 4.458 10 10 10s10-4.458 10-10S15.542 0 10 0Z"/><path d="M17.042 10c0 3.875-3.166 7.041-7.041 7.041-3.875 0-7.042-3.166-7.042-7.041 0-3.875 3.125-7 7-7.042v-.75c-4.292.042-7.75 3.542-7.75 7.833.042 4.292 3.542 7.75 7.833 7.75 4.292-.041 7.75-3.5 7.75-7.791h-.75Z"/><path d="M10 4.167a5.87 5.87 0 0 0-4.125 1.708l.5.5c2.042-1.958 5.292-1.833 7.25.208 1.875 1.958 1.875 5.042 0 7.042l.5.5a5.81 5.81 0 0 0 0-8.25A5.87 5.87 0 0 0 10 4.167Zm3.625 9.458.5.5-.5-.5Zm-7.75-7.75.5.5-.5-.5Z"/></g><defs><clipPath id="a"><path fill="#fff" d="M0 0h20v20H0z"/></clipPath></defs></svg>
```
Comment (ItsANameToo) [inline arkscan/icons/bitladon.svg]: id

**Review by ItsANameToo — CHANGES_REQUESTED** [review]
@alfonsobries there are still a couple SVGs that rely on short `ids` that need adjusting to avoid clashes, otherwise the `url` reference picks whatever is last and results in weird styles

**Comment by alfonsobries** [comment]
@ItsANameToo updated with a more unique id

**Review by ItsANameToo — APPROVED** [review]

**Pull request merged**